README.md

```diff
@@ -1,23 +1,115 @@
 # bazel
 
-Bazel support for Dart
+[Bazel][bazel] extension support for [Dart][dart].
 
-**WARNING**: This package is highly experimental.
+Bazel is a _correct, reproducible, and fast_ build tool used internally at
+Google, and now open source, that provides a powerful and extensible framework
+for building software and maintaining code.
 
-To use, create a directory structure _similar_ to `workspace`:
+[bazel]: https://www.bazel.io/
+[dart]: https://www.dartlang.org/
 
+## Why Bazel
+
+Large applications in Google like [AdWords Next][blog-awn] and
+[AdSense](blog-ads) have used Dart and [Angular Dart][angular-dart] in
+production with Bazel for a while, and now we want to share Bazel with the rest
+of the community. 
+
+[blog-awn]: http://news.dartlang.org/2016/03/the-new-adwords-ui-uses-dart-we-asked.html
+[blow-ads]: http://news.dartlang.org/2016/10/google-adsense-angular-dart.html
+[angular-dart]: https://angular.io/dart
+
+
+## Getting Started
+
+> **WARNING**: This package is highly experimental. While the underlying
+> framework (such as BUILD rules and extensions) are stable, we're still
+> iterating on a good stable solution for end users that are used to primarily
+> working with pub and `pubspec.yaml` files.
+>
+> Have a suggestion to make this better? [File an issue][file_issue].
+
+[file_issue]: https://github.com/dart-lang/bazel/issues/new
+
+Our Bazel package publishes a `bazelify` command that takes an existing pub
+package and automatically generates a Bazel worskpace: full of extensions,
+macros, rules, and more.
+
+### Installation
+
+If you're familiar with [`pub run`][pub_run], then `bazelify` is easy:
+
+[pub_run]: https://www.dartlang.org/tools/pub/cmd/pub-run
+
+```bash
+$ pub global activate bazel
+```
+
+### Generation
+
+You can run `bazelify` on a typical `pub` package:
+
+```
+my_new_package/
+  bin/
+  lib/
+  web/
+  pubspec.yaml
+```
+
+```bash
+pub global run bazel:bazelify -p my_new_package
```

Comment:
is `-p my_new_package` necessary if run from within a package dir?

Perhaps

```bash
my_pkg_dir/ $ pub global run bazelify
```

Note, you don't need `bazel:` for globally activated binaries